prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -47,6 +47,9 @@ gke_minio_gateway_job_template: &gke_minio_gateway_job_template
         privileged: true
       command:
       - "bash"
+      env:
```

Comment:
Can't we put this information in a preset and reuse from there? This way, if we need to change it in the future, it'd be easier and more reliable to change it from only one place.

Reply:
Thst would be preset with one variable and we would have to have preset for each zone (europe-west4-a, europe-west4-b, europe-west4-c). With new setup it is clear which variables are passed to the job.